content/chainguard/chainguard-images/working-with-images/scanners/trivy-tutorial/index.md

```diff
@@ -0,0 +1,321 @@
+npm init -y
+npm install qs@6.5.2
+trivy fs .
+```
+
+You should see a summary and itemized list of CVEs for the outdated Node package.
+
+#### Scanning Clusters
+
+To scan a Kubernetes cluster:
+
+```bash
+trivy k8s --report summary <cluster-name>
+```
+
+To try out the above command on a test Kubernetes cluster, First install [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation), a utility allowing Kubernetes to be run on your local machine.
+
+Once Kind is installed and accessible on your path, run the following to create a cluster:
+
+```bash
+kind create cluster --name test-cluster
+```
+
+Run the following to scan the new cluster with Trivy:
+
+```bash
+trivy k8s --report summary kind-test-cluster
+```
+
+When scanning clusters, requesting only summary output is recommended, as tables in more verbose output may not display correctly.
+
+### Scanning SBOMs
+
+Trivy can both generate and scan SBOMs.
+
+{{< details "What is an SBOM?" >}}
+{{< blurb/sbom >}}
+{{< /details >}}
+
+First, generate an SBOM file in CycloneDX format to scan
+
+```bash
+trivy image -f cyclonedx -o results.cdx.json nginx
+```
+
+Trivy can scan this generated CycloneDX SBOM with the following:
+
+```bash
+trivy sbom results.cdx.json
+```
+
+By default, the `sbom` subcommand scans only for vulnerabilities. License scanning can be enabled using the `--scanners license` flag. 
+
+Learn more on SBOMs and other output formats in the section on [specifying output formats](#specifying-output-formats).
+
+## Comprehending Trivy Output
+
+When run with default output and formatting, Trivy first prints a series of informational messages and warnings, then the name of the image and a one-line summary of the number and severity of issues found, and finally a table itemizing each issue.
+
+ In this section, we'll use an Alpine version of the official Python image as an example. Since we're specifying an older version, you may encounter more CVEs when following the examples than are shown here, as CVEs will accumulate on an image over time.
+
+Scan the image with the following command:
+
+```bash
+trivy image python:3.10.14-alpine3.20
+```
+
+You will receive output similar to the following:
+
+![Screenshot of Trivy output showing first informational messages on which scanners are inabled, then the name of the image, then a listing of issues by severity, and finally a table formatted with ASCII seperators (pipes and underscores) showing itemized issues](trivy_output.png)
+
+### Interpreting Trivy Output
+
+The initial logging portion of Trivy's output indicates which [scanners](#scanners) are enabled and shows warnings if Trivy has an issueperforming the scan. 
+
+In the initial portion of its results output, Trivy summarizes information on the scanned artifact and gives an overview of known vulnerabilities. In the case of a scanned image, the output includes the image digest, a unique hash of the image that can be used as an identifier. 
```

Comment:
I wonder if the last comma should be ";" so it doesn't read like a list.

Reply:
Made a new sentence.